rootfs/etc/nginx/lua/util/nodemap.lua

```diff
@@ -0,0 +1,120 @@
+local math = require("math")
```

Comment:
same as https://github.com/kubernetes/ingress-nginx/pull/4514/files#r324827563, you only use `random` function from this module